reports/assist-mini/2024.8.0dev/reports.yaml

```diff
@@ -3,22 +3,10 @@
   good_percent: 63.3%
```

Comment:
Let's revert existing report file changes that are just sorting the yaml files.

Is this some kind of auto yaml sorter pre-commit or a vscode plugin?

Reply:
the prebuild script did this for a reason i didn't quite understand. i figured there's a reason for it, so i went with it. i'll revert

Reply:
OK thanks it may be random. i'll update to make it sorted or something consistent.